docs-ref-conceptual/eclipse/java-sdk-azure-get-started-eclipse.md

```diff
@@ -0,0 +1,419 @@
+---
+title: Get started with Azure for Java using Eclipse
+description: Get started with basic use of the Azure libraries for Java with your own Azure subscription.
+keywords: Azure, Java, SDK, API, authenticate, get-started
+author: roygara
+ms.author: v-rogara
+manager: timlt
+ms.date: 10/30/2017
+ms.topic: get-started-article
+ms.prod: azure
+ms.technology: azure
+ms.devlang: java
+ms.service: multiple
+---
+
+# Get started with the Azure using Eclipse
+
+This guide walks you through setting up a development environment with an Azure service principal. It also provides you with sample code that creates and uses resources in your Azure subscription using the Azure libraries for Java.
+
+## Prerequisites
+
+- An Azure account. If you don't have one, [get a free trial](https://azure.microsoft.com/free/)
+- [Azure Cloud Shell](https://docs.microsoft.com/azure/cloud-shell/quickstart) or [Azure CLI 2.0](https://docs.microsoft.com/cli/azure/install-az-cli2).
+- [Eclipse](http://www.eclipse.org/downloads/)
+
+## Set up authentication
+
+Your Java application needs read and create permissions in your Azure subscription to run the sample code in this tutorial. Create a service principal and configure your application to run with its credentials. Service principals provide a way to create a non-interactive account associated with your identity to which you grant only the privileges your app needs to run.
```

Comment:
Create a service principal  (link to doc) to grant your code permission to create and update resources in your subscription without using your account credentials directly.